### PR TITLE
[JENKINS-38928] Fix apply behaviour when form contains radio group

### PR DESCRIPTION
**Pull request by Vlatombe**
https://issues.jenkins-ci.org/browse/JENKINS-38928

This fixes the form handling issue after 'Apply' is clicked.

@reviewbybees

### CODE REVIEW

**Comment by Vlatombe** [comment]
ping @olivergondza in case you have any opinion about this.

**Comment by olivergondza** [comment]
I am not able to reason about the fix. The problem seems reproducible in ATH to verify if it gets fixed without breaking stuff.

**Comment by Vlatombe** [comment]
good point, i'll add a test in ATH for this

**Comment by Vlatombe** [comment]
@olivergondza look like I had misunderstood the method logic and the one in form-element-path does build a different thing (despite the similar structure). So now we are down to a single line fix ;)

I wrote an ATH test (https://github.com/jenkinsci/acceptance-test-harness/pull/210) that this PR is fixing.

**Comment by Vlatombe** [comment]
@olivergondza I suggest to merge this PR first, release then update https://github.com/jenkinsci/acceptance-test-harness/pull/210 with ref to the new plugin version to avoid failing ATH in the mean time.

**Comment by olivergondza** [comment]
@Vlatombe, that would require you pushing alpha release again. How about creating Jenkinsfile for this plugin to run ATH with PR changes? It will verify we have not break something accidentally.

**Comment by Vlatombe** [comment]
@olivergondza Just took a first stab at this, however jenkins.ci is running some too old plugins at the moment to have it running properly. Will work on getting this fixed.

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
+def formElementPathVersion
+node('hi-speed') {
+    stage('plugin') {
+        dir('plugin') {
+            checkout scm
+            timeout(30) {
+                withMaven(jdk: 'Oracle JDK 1.8 (latest)',
+                        maven: 'Maven 3.2.1',
+                        mavenSettingsConfig: 'org.jenkinsci.plugins.configfiles.maven.MavenSettingsConfig1382087369423',
```
Comment (olivergondza) [inline Jenkinsfile]: Does this refer to particular instance config? Why?

Comment (Vlatombe) [inline Jenkinsfile]: This refers to https://jenkins.ci.cloudbees.com but it probably deserves a human-readable id.

Comment (olivergondza) [inline Jenkinsfile]: Is there a way to configure the pom.xml to be independent on Jenkins config (such as global setting.xml)?

Comment (Vlatombe) [inline Jenkinsfile]: I think it is already, I can probably drop this option.

**Comment by olivergondza** [comment]
ci.jenkins.io does not seem to pick it up. I pushed your Jenkinsfile work to master to get it built but I guess it needs to be re-scanned: https://ci.jenkins.io/job/Plugins/computation/

**Comment by Vlatombe** [comment]
I'm not sure ci.jenkins.io is set up to execute ATH. I have set up the pipeline on https://jenkins.ci.cloudbees.com/job/experiments/job/vlatombe/job/form-element-path/ for now and it seems to run fine.

**Pull request merged**